server.js

```diff
@@ -20,8 +21,61 @@ app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+app.get('/', (req, res) => {
+  res.send('Welcome to my book API!');
+});
+
+//endpoint to get one random book
+app.get('/books/random', (req, res) => {
+  const randomBook = Data[Math.floor(Math.random() * Data.length)];
+  randomBook.length !== 0
+    ? res.json(randomBook)
+    : res.status(404).send('No books found, please try again'); //if no books are found, send error message (this should not happen...)
+});
+
+// Endpoint to get all books in a specific language
+app.get('/books/language/:lang', (req, res) => {
+  const lang = req.params.lang;
+  const booksInLanguage = Data.filter((item) => item.language_code === lang);
+
+  booksInLanguage.length !== 0
+    ? res.json(booksInLanguage)
+    : res.status(404).send('No books in that language found, please try again'); //if no books are found, send error message
+});
+
+// Endpoint to get all books with a specific rating and in a specific language
+app.get('/books/language/:lang/rating/:rating', (req, res) => {
```

Comment:
Is there a way to be able to round the numbers up and use no decimals? It makes it pretty hard for the user/developer to come up with a search for "4.56" with no info about those ratings beforehand.